Makefile

```diff
@@ -64,6 +64,7 @@ unit-test-nocoverage-host: golint-host govet-host
 
 build: golint govet
 	vagrant ssh mon0 -c 'sudo -i sh -c "cd $(GUESTGOPATH); make run-build"'
+	make run
```

Comment:
Yeah, `make build` is run by both the systemtests and `make start`, so the `run` bits will get applied at the right times.

The systemtests already accounted for this, but I validated that first.